src/components/ImportExport.js

```diff
@@ -36,24 +56,39 @@ class ImportExport extends React.Component {
     });
   }
 
+
   importFile = () => {
     const { selectedFiles } = this.state;
     const { importKMLClicked, trails, hydrants } = this.props;
 
     function processTrail(feature, index) {
-      let [name, ...otherThings] = feature.get('description').split(',');
+      let [name, ...otherThings] = feature.get('description').split(',') ;
       name = _.words(name).join(' ');
       const coords = feature.getGeometry().getCoordinates()[0];
       const lonLatCoords = _.map(coords, pt => Projection.fromLonLat(pt.slice(0, 2)));
+
+      const featureFill = feature.getStyle().call(feature)[0].getFill()
+
+      // this needs to be just rgb sepeated
+      const fillColor = featureFill ? featureFill.getColor().slice(0, 3).join(',') : '255,255,255'
```

Comment:
This will break if the import comes in as a hex color. You could see if the format looks like a hex and convert it to rgb instead (simple function you can paste in utils somewhere, just google it).

Reply:
Check